docs/asciidoc/modules/ROOT/pages/temporal/datetime-conversions.adoc

```diff
@@ -273,7 +273,7 @@ This function parses a date string in a specified format and converts it to a Ne
 
 [NOTE]
 ====
-This function has been renamed to `apoc.temporal.toZonedDateTime` and moved to the temporal section.
+This function has been renamed to `apoc.temporal.toZonedTemporal` and moved to the temporal section.
```

Comment:
Maybe this could be rephrased so it is even clearer that`apoc.date.parseAsZonedDateTime` function is deprecated and `apoc.temporal.toZonedTemporal` is what should be used instead?

Reply:
Changed